src/contexts/ConfigContext/reducer.tsx

```diff
@@ -0,0 +1,52 @@
+import {
+  ConfigStateType,
+  ConfigDispatchTypeEnum,
+  ConfigDispatchActionType
+} from './types';
+
+const reducer = (state: ConfigStateType, action: ConfigDispatchActionType) => {
```

Comment:
you can simply write export const.. but it's OK like this also

Reply:
updated